docs/DeveloperGuide.md

```diff
@@ -890,6 +1073,47 @@ testers are expected to do more *exploratory* testing.
 
 
+---
+
+## Appendix: Efforts
+
+WedLinker is a project adapted from the original [AddressBook3 (AB3)](https://se-education.org/addressbook-level3/) customized specifically for wedding planners by our team of five.
+Throughout this project, we had split the work evenly and adhering to the principles and concepts that were taught by the
+CS2103T teaching team, including:
+- [Forking Workflow](https://nus-cs2103-ay2425s1.github.io/website/schedule/week7/topics.html#project-management-revision-control-forking-flow)
+- [Feature Branch Flow](https://nus-cs2103-ay2425s1.github.io/website/schedule/week7/topics.html#project-management-revision-control-feature-branch-flow)
+- [UML Class Diagrams](https://nus-cs2103-ay2425s1.github.io/website/se-book-adapted/chapters/uml.html#class-diagrams)
+- [Testing](https://nus-cs2103-ay2425s1.github.io/website/se-book-adapted/chapters/testing.html)
+
+As we envisioned an application for wedding planners, we concocted multiple features based on feedback from the wedding planners we interviewed.
+These are the features that shape the WedLinker you see today.
+
+#### Difficulty Level:
+Adapting from the original AB3 to the WedLinker has been difficult as we opted for a more daring approach to implement 
+many features that wedding planners wanted. As with any software, the more features implemented, the more bugs there are,
+and WedLinker was no exception to it. 
+
+As a result, the difficulty of developing WedLinker came not only from the
+myriad of features we had to implement (and alter from the original AB3), but to also ensure that there were minimal bugs created.
+
+#### Challenges Faced:
+Undoubtedly with any projects, there will be challenges faced. 
+With the number of features that we wanted to implement, we had used the [Forking Workflow](https://nus-cs2103-ay2425s1.github.io/website/schedule/week7/topics.html#project-management-revision-control-forking-flow)
+that enabled us to work on numerous features, debugs, or minor adjustments concurrently. However, with the Forking Workflow came some drawbacks that we personally experienced, such as
+the extra overhead of sending everything through forks, as well as the increased possibility of merge conflicts.
+
+#### Effort Required:
+As we had chosen to implement numerous features and inevitably introduced bugs per iteration, the effort required to
+resolve these issues were higher than if we had opted to be safe and tweak AB3 just a little.
+
+#### Achievements:
+As of 12 Nov 2024, WedLinker has:
+
+- [Most LoC (lines of code) in AY2425 S1 CS2103T](https://nus-cs2103-ay2425s1.github.io/tp-dashboard/?search=&sort=totalCommits%20dsc&sortWithin=title&timeframe=commit&mergegroup=AY2425S1-CS2103T-F15-4%2Ftp%5Bmaster%5D&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2024-09-20) at 21284
+- **20** new commands as compared to the original AB3
```

Comment:
I think it might be 17, I might be missing some?:

The list I have is:
1. create tag
2. delete tag
3. list tags
4. assign task
5. create task
6. delete task
7. unassign task
8. mark task
9. unmark task
10. assign vendor
11. unassign vendor
12. assign wedding
13. unassign wedding
14. create wedding
15. delete wedding
16. edit wedding
17. list weddings

Reply:
list tasks
tag
untag

Reply:
You're right, thanks for specifying